src/utils/toast/toast.scss

```diff
@@ -0,0 +1,7 @@
+.custom-toast {
+  display: flex !important;
```

Comment:
guess there was not a better choice

Reply:
No sir. sweetalert styles block another specified properties